lib/pages/contacts_tab/contacts_appbar_style.dart

```diff
@@ -1,24 +1,40 @@
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/utils/responsive/responsive_utils.dart';
 import 'package:flutter/cupertino.dart';
 
 class ContactsAppbarStyle {
+  static ResponsiveUtils responsiveUtils = getIt.get<ResponsiveUtils>();
```

Comment:
not a good idea, please inject it in method. Fx: `static AlignmentGeometry alignmentTitle(responseUtils, context)`

Reply:
applied as other style file